api/src/main/scala/WebServer.scala

```diff
@@ -9,9 +9,19 @@ import DefaultJsonProtocol._
 import com.typesafe.config.ConfigFactory
 import com.typesafe.scalalogging.LazyLogging
 
-import geotrellis.raster.NODATA
+case class InputData(operationType: String, rasters: List[String], zoom: Int,
+  polygonCRS: String, rasterCRS: String, polygon: List[String])
+
+case class PostRequestJson(input: InputData)
```

Comment:
Since `json` is one serialization of the post request, but the `PostRequest` object itself can exist in many forms (including the Scala native form used herein), consider renaming `PostRequestJson` to `PostRequest`.